packages/odyssey-storybook/src/components/odyssey-mui/Select/Select.stories.tsx

```diff
@@ -527,3 +537,42 @@ export const ControlledPreselectedMultipleSelect: Story = {
     return <Select {...props} value={localValue} onChange={onChange} />;
   },
 };
+
+export const MultipleLanguages: Story = {
+  args: {
+    label: "Select Language",
+    options: optionsLanguages,
+    defaultValue: "",
+  },
+  parameters: {
+    docs: {
+      description: {
+        story:
+          "Showcase the Select component with options for multiple different languages.",
+      },
+    },
+  },
+  play: async ({ canvasElement, step }) => {
+    await step("Select English from the listbox", async () => {
+      const comboBoxElement = canvasElement.querySelector(
+        '[aria-haspopup="listbox"]',
+      );
+      if (comboBoxElement) {
+        await userEvent.click(comboBoxElement);
+        const listboxElement = screen.getByRole("listbox");
+        await expect(listboxElement).toBeInTheDocument();
+        const listItem = listboxElement.children[0];
+        await userEvent.click(listItem);
+        await userEvent.tab();
+        await waitFor(() => {
+          expect(listboxElement).not.toBeInTheDocument();
+        });
+        const selectedOption = listboxElement.querySelector('[lang="fr"]');
```

Comment:
This part of the test seems backward and not how a user would interact with the select.

We want to select the French option by label (what the user sees and hears) and then check to see if the `lang="fr"` attribute is on there.

```suggestion
        const selectedOption = listboxElement.querySelector('[lang="fr"]');
```

We can also use testing library selectors here:
```ts
const frenchOptionElement = within(listboxElement).getByRole("option", { label: "Français" })

expect(frenchOptionElement).toHaveAttribute("lang", "fr")
```

Reply:
updated with `within(listboxElement).getByText("Français");`